pages/sticky-notes.html

```diff
@@ -13,79 +13,75 @@
+                <ul>
+                    <button class="global__menu__close-btn"><img src="../images/close-btn.svg" alt=""></button>
+                    <a href="../index.html"><img src="../images/logo.png" alt="" class="dashboard-logo"></a>
+                    <li><img src="../images/global-menu.svg" alt=""><a href="../pages/dashboard.html">Dashboard</a></li>
+                    <li><img src="../images/todo.svg" alt=""><a href="../pages/to-do.html">Todo List</a></li>
+                    <li><img src="../images/notes.svg" alt=""><a href="../pages/sticky-notes.html">Sticky Notes</a></li>
+                    <li><img src="../images/pomodoro.svg" alt=""><a href="../pages/pomodoro.html">Pomodoro</a></li>
+                    <li><img src="../images/timer.svg" alt=""><a href="../pages/time-tracker.html">Timer</a></li>
+                    <li><img src="../images/gift.svg" alt=""><a href="../pages/additional.html">Additional</a></li>
+                </ul>
             </div>
-            <div class="nav__right__user-profile">
-                <p class="nav__right__user-profile__user-name">Jasleen</p>
-                <img src="../images/arrow-down.svg" alt="" class="nav__right__user-profile__dropdown-btn">
+            <a href="../index.html"><img src="../images/logo.png" alt="" class="dashboard-logo dashboard-desktop-logo"></a>
+            <div class="profile">
+                <p class="profile-user">Jasleen</p>
+                <img src="../images/arrow-down.svg" alt="" class="profile-dropdown">
+                <ul class="profile-open">
+                    <button class="global__menu__close-btn global__menu__close-btn-profile"><img src="../images/close-btn.svg" alt=""></button>
+                    <li><img src="../images/settings.svg" alt=""><a href="../pages/settings.html">Settings</a></li>
+                    <li><img src="../images/logout.svg" alt=""><a href="">Logout</a></li>
+                </ul>
             </div>
-        </div>
-
         </nav>
     </header>
-    <h1 class="sticky__heading">Sticky Notes</h1>
+        <main class="dashboard-main">
+     
+                <ul class="side-bar">
+                    <li><img src="../images/global-menu.svg" alt=""><a href="../pages/dashboard.html">Dashboard</a></li>
+                    <li><img src="../images/todo.svg" alt=""><a href="../pages/to-do.html">Todo List</a></li>
+                    <li><img src="../images/notes.svg" alt=""><a href="../pages/sticky-notes.html">Sticky Notes</a></li>
+                    <li><img src="../images/pomodoro.svg" alt=""><a href="../pages/pomodoro.html">Pomodoro</a></li>
+                    <li><img src="../images/timer.svg" alt=""><a href="../pages/time-tracker.html">Timer</a></li>
+                    <li><img src="../images/gift.svg" alt=""><a href="../pages/additional.html">Additional</a></li>
+                </ul>
+
+                <div class="main-content">
+                    
+    <div id="dashboard-root">
```

Comment:
Why the formatting is messed up?

Have you installed ESLint extension on VSCode? It usually automatically fixes the formatting issues.

Reply:
Yes, I've installed ESLint extension. I'll look into the matter & solve it.